server/embed/config.go

```diff
@@ -216,10 +224,14 @@ type Config struct {
 	DNSCluster            string `json:"discovery-srv"`
 	DNSClusterServiceName string `json:"discovery-srv-name"`
 	Dproxy                string `json:"discovery-proxy"`
-	Durl                  string `json:"discovery"`
-	InitialCluster        string `json:"initial-cluster"`
-	InitialClusterToken   string `json:"initial-cluster-token"`
-	StrictReconfigCheck   bool   `json:"strict-reconfig-check"`
+
+	EnableV2Discovery bool   `json:"enable-v2-discovery"`
+	Durl              string `json:"discovery"`
+	DiscoveryCfg      v3discovery.DiscoveryConfig
```

Comment:
Please add `json` annotation to discovery

Reply:
Actually previously I thought about it, but I noticed that fields, such as ClientTLSInfo and PeerTLSInfo, do not have json annotations, so I did not add the annotation either. 

Anyway, I will add the json annotation.

Reply:
Thanks & fixed.